class/trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    trimmed_name = @nameable.correct_name[0..10]
+    @nameable.correct_name.length >= 10 ? trimmed_name : @nameable.correct_name
+  end
```

Comment:
- [ ] This is a close one as well but the indices you've provided in line `5` will trim the words to 11 characters instead of 10 so you should instead use `@nameable.correct_name[0..9]` (remember the counting starts from 0 😅).